app/src/main/java/com/mybus/activity/MainActivity.java

```diff
@@ -568,18 +581,10 @@ public void onRoadFound(RoadResult roadResult) {
 
     /**
      * Populates the bottom sheet with the Routes found
-     *
-     * @param results
      */
-    private void populateBottomSheet(List<BusRouteResult> results) {
-        if (results == null || results.isEmpty()) {
-            showBottomSheetResults(false);
-            mViewPagerAdapter = null;
-            Toast.makeText(this, R.string.toast_no_result_found, Toast.LENGTH_LONG).show();
-            return;
-        }
+    private void populateBottomSheet(int busResultId) {
```

Comment:
Revert to the original version which receives the List<BusRouteResult> and just add the busRestultId